src/components/list/list-base.scss

```diff
@@ -0,0 +1,300 @@
+/** External component calculations */
+$md-checkbox-width: 3 * $baseline-grid !default;
+//TODO(devversion): use shared variable from buttons.scss
+$md-button-spacing: $button-left-right-padding + rem(0.800) !default;
+
+/** Static List Item Variables */
+$md-list-item-indention: $baseline-grid * 7 !default;
+$md-list-item-padding: $baseline-grid * 2 !default;
+$md-list-item-h3-margin: 0 0 0px 0 !default;
+$md-list-item-h4-margin: 3px 0 1px 0 !default;
+$md-list-item-h4-font-weight: 400 !default;
+$md-list-item-p-margin: 0 0 0 0 !default;
+$md-list-item-subheader-height: 6 * $baseline-grid !default;
+
+/** Resets the checkbox component */
+@mixin md-reset-checkbox() {
+  md-checkbox {
+    align-self: center;
+    margin: 0;
+
+    .md-label {
+      display: none;
+    }
+  }
+}
+
+/** Resets the switch component to properly align */
+@mixin md-reset-switch() {
+  md-switch {
+    margin-top: 0;
+    margin-bottom: 0;
+
+    @include rtl-prop(margin-right, margin-left, -6px);
+  }
+}
+
+/** Reset browsers default paragraph styles */
+@mixin md-reset-paragraph() {
+  p {
+    margin: 0;
+    flex: 1;
+  }
+}
+
+/** Reset default md-button behavior like uppercase and text alignment */
+@mixin md-reset-button() {
+  &.md-button {
+    font-size: inherit;
+    text-transform: none;
+
+    @include rtl(text-align, left, right);
+
+    &, > .md-ripple-container {
+      border-radius: 0;
+      margin: 0;
+      padding: 0;
+    }
+  }
+}
+
+/** Mixin to overwrite the size of an md-icon element */
+@mixin md-icon-size($icon-size) {
+  min-width: $icon-size;
+  min-height: $icon-size;
+  height: $icon-size;
+  width: $icon-size;
+
+  svg {
+    width: $icon-size;
+    height: $icon-size;
+  }
+}
+
+/** Creates an overlaying button, which will be used to execute the primary action */
+@mixin md-button-wrap() {
+  &.md-button-wrap {
+    position: relative;
+
+    .md-button-wrap-executor {
+      position: absolute;
+      top: 0;
+      left: 0;
+
+      height: 100%;
+      width: 100%;
+
+      @include md-reset-button();
+    }
+  }
+}
+
+/**
+ * Mixin to style user specified content inside of the list item.
+ */
+@mixin md-list-item-content($_list-icon-width, $_list-avatar-width, $_list-avatar-padding-dense) {
```

Comment:
As we talked, I don't really like the `$_` prefix for parameters.. maybe local variables should have it instead?